packages/server/test/support/fixtures/projects/e2e/cypress/integration/headless_spec.js

```diff
@@ -1,8 +1,62 @@
+const { _ } = Cypress
 const expectedHeadless = !!Cypress.env('EXPECT_HEADLESS')
 
 describe('e2e headless spec', function () {
   it('has the expected values for Cypress.browser', function () {
     expect(Cypress.browser.isHeadless).to.eq(expectedHeadless)
     expect(Cypress.browser.isHeaded).to.eq(!expectedHeadless)
   })
+
+  it('has expected HeadlessChrome useragent', function () {
+    if (Cypress.browser.family !== 'chrome') {
+      return
+    }
+
+    cy.wrap(navigator.userAgent)
+    .should(expectedHeadless ? 'contain' : 'not.contain', 'HeadlessChrome')
+  })
+
+  it('has expected launch args', function () {
+    if (Cypress.browser.family !== 'chrome') {
+      return
+    }
+
+    cy.task('get:browser:args')
+    .should(expectedHeadless ? 'contain' : 'not.contain', '--headless')
+  })
+
+  it('has expected window bounds in CI', function () {
+    if (!Cypress.env('CI')) {
+      // will only work in the CI docker container, so skip it if we're not in CI
+      return this.skip()
+    }
+
+    if (Cypress.browser.family !== 'chrome') {
+      // Browser.getWindowForTarget does not exist in Electron
+      return
+    }
+
+    // in CI with Xvfb, Cypress will fill the entire screen, this test is to explicitly
+    // assert on the expected dimensions
+
+    const cdp = _.bind(Cypress.automation, Cypress, 'remote:debugger:protocol')
+
+    return cdp({
+      command: 'Browser.getWindowForTarget',
+    })
+    .then(({ windowId }) => {
+      return cdp({
+        command: 'Browser.getWindowBounds',
+        params: {
+          windowId,
+        },
+      })
+    })
+    .then(({ bounds }) => {
+      expect(bounds).to.include({
+        width: expectedHeadless ? 800 : 1050,
+        height: expectedHeadless ? 600 : 1004,
```

Comment:
800x600 is the expected dimensions when running headlessly in CI?

Reply:
yes, that seems to be the reported "window bounds" when running with `--headless`

Reply:
it seems to be 800x600 irrespective of where you ran it - i get the same thing running outside of the docker container

Reply:
This is really weird... maybe leave a comment that this is expected behavior. It makes me think we shouldn't even be writing this test this way - it communicates kind of the wrong thing, and your other two tests already pretty much guarantee that `headless` is correct.

Reply:
Yeah, the only value this test really adds is to give us a warning if Chrome or Xvfb change their behavior. 

There is already a comment, should I add something to it?

```
    // in CI with Xvfb, Cypress will fill the entire screen, this test is to explicitly
    // assert on the expected dimensions
```

Reply:
I would probably just remove the test for now

Reply:
![deleted](https://user-images.githubusercontent.com/1151760/70921098-a55aeb80-1ff1-11ea-910d-84525f14d3df.png)